base64ct/src/errors.rs

```diff
@@ -15,29 +15,17 @@ impl fmt::Display for InvalidLengthError {
 #[cfg(feature = "std")]
 impl std::error::Error for InvalidLengthError {}
 
-/// Invalid encoding of provided "B64" string.
-#[derive(Copy, Clone, Debug, Eq, PartialEq)]
-pub struct InvalidEncodingError;
```

Comment:
There are no longer any cases where this is useful as a separate error anymore, as `decode_in_place` now needs to handle the case where the input is padded but the wrong length.

Reply:
I think wrong padding length falls under `InvalidEncodingError`, not under `InvalidLength`.

Reply:
Okay, but there are still no functions in the public API that return `InvalidEncodingError`, so I don't think it's worth having an extra type for it.

If you really want to keep it around, I think it should be private for now.

Reply:
Added back `InvalidEncodingError` and used it for `decode_in_place` again